airbyte-webapp/src/services/job/JobService.tsx

```diff
@@ -25,8 +25,9 @@ function useGetJobService() {
 
 export const useListJobs = (listParams: JobListRequestBody) => {
   const service = useGetJobService();
-  return useSuspenseQuery(jobsKeys.list(listParams.configId), () => service.list(listParams), {
+  return useSuspenseQuery(jobsKeys.list(listParams.configId, listParams.pagination), () => service.list(listParams), {
     refetchInterval: 2500, // every 2,5 seconds,
+    keepPreviousData: true,
```

Comment:
This was necessary to prevent the jobs from disappearing and the loading spinner showing every time `Load more` is clicked. With this keepPreviousData option set to true, it now seamlessly appends the new query results to the end once they are available. For more info see the official docs on this: https://react-query-v3.tanstack.com/guides/paginated-queries

Reply:
This is a really cool solution for this.